README.md

```diff
@@ -1 +1 @@
-testing semantic versioning 4.0.0
+testing semantic versioning 4.1.0
```

Comment:
- It would be beneficial to provide a brief description of the changes made in the commit message for better understanding.